extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -32,24 +34,54 @@ public function getRssFeeds() {
 				return;
 			}
 
-			$status = null;
-			$rss = @fetch_rss($url, $status);
+			$feedData = WikiaDataAccess::cacheWithLock(
+				wfSharedMemcKey( 'WikiaRss', 'feed', $url ),
```

Comment:
When we move a method into other place it usually involves changing the logic. `__METHOD__` acts as an automatic cache versioning ;)